ui/src/hooks/useSharedConfig.ts

```diff
@@ -5,28 +5,33 @@ import { StarterRequest } from '../api/starter';
 import { SnackbarConfig } from '../components/CommonSnackbar';
 import { useSearchParams } from 'react-router-dom';
 
-export function useSharedConfig(): [StarterRequest | undefined, SnackbarConfig | undefined, boolean] {
+export function useSharedConfig(): [StarterRequest | undefined, SnackbarConfig | undefined, boolean, boolean] {
   const [searchParams, setSearchParams] = useSearchParams();
   const [request, setRequest] = useState<StarterRequest>();
   const [snackbar, setSnackbar] = useState<SnackbarConfig>();
   const [ready, setReady] = useState(false);
+  const [preview, setPreview] = useState(false);
 
   useEffect(() => {
     if (searchParams.toString() === '') {
       setReady(true);
       return;
     }
     const params = parse(searchParams.toString(), { parseBooleans: true });
+    const shouldPreview = 'preview' in params;
+    setPreview(shouldPreview);
     starterValidationSchema
       .isValid(params)
       .then(isValid => {
         if (isValid) {
           setRequest(params as StarterRequest);
-          setSnackbar({
-            open: true,
-            severity: 'info',
-            message: 'Linked configuration was applied.',
-          });
+          if (!shouldPreview) {
+            setSnackbar({
+              open: true,
+              severity: 'info',
+              message: 'Linked configuration was applied.',
```

Comment:
AFAIK it is a good practice to separate the message's content from the code. Perhaps there is some component in the project acting like messages repository and this message should be placed there? If not, please ignore this suggestion.

Reply:
I would say that we should discuss it as a separate issue for `adopt-tapir` app - if there are any plans to i18n it.

Reply:
No plans ATM